src/ducks/SANCharts/utils.js

```diff
@@ -34,7 +34,10 @@ export const Metrics = {
   dailyActiveAddresses: {
     node: Bar,
     color: 'malibu',
-    label: 'Daily Active Addresses'
+    label: 'Daily Active Addresses',
+    dataKey: 'active_addresses',
+    orientation: 'right',
```

Comment:
Is it critical changes for our /charts ?

Reply:
Crytical. Will change to custom active_addresses back

Reply:
Changed